in_toto/model.go

```diff
@@ -31,6 +33,24 @@ type Key struct {
 	Scheme              string   `json:"scheme"`
 }
 
+func validateKeyIdFormat(keyId string) error {
```

Comment:
Minor nit: I'd omit the `Format`-suffix, because you don't have it anywhere else.